package.json

```diff
@@ -144,6 +144,7 @@
     "libxmljs": "^0.19.5",
     "metalsmith-assets": "^0.1.0",
     "metalsmith-broken-link-checker": "^1.0.1",
+    "metalsmith-date-in-filename": "^0.0.14",
```

Comment:
I moved this to dev dependencies because it doesn't need to be run on the preview server, and that's the only time Metalsmith is executed live.